linkerd/app/core/src/svc.rs

```diff
@@ -228,13 +230,13 @@ impl<S> Stack<S> {
         self.push_on_service(buffer(name, config))
     }
 
-    pub fn push_cache<T>(self, idle: Duration) -> Stack<cache::NewCachedService<T, S>>
+    pub fn push_cache<T>(self, idle: Duration) -> Stack<idle_cache::NewIdleCached<T, S>>
```

Comment:
nit, take it or leave it: should this maybe be called `push_idle_cache`? that way it's more obvious which cache implementation is being pushed...